README.md

```diff
@@ -46,7 +48,7 @@ found in Libursa.
 Ursa and zmix use the following external dependencies:
 
 - [libsodium
-  1.0.14](https://download.libsodium.org/libsodium/releases/old/libsodium-1.0.14.tar.gz)
+  1.0.14](https://download.libsodium.org/libsodium/releases/old/libsodium-1.0.16.tar.gz)
```

Comment:
Do you want to update these references too?
 (mikelodder7-travis-ci)$ grep -r libsodium-1.0.14.tar.gz *
docker/centos/Dockerfile:    && curl -fsSL https://download.libsodium.org/libsodium/releases/old/libsodium-1.0.14.tar.gz | tar -xz \
docker/fedora/Dockerfile:    && curl -fsSL https://download.libsodium.org/libsodium/releases/old/libsodium-1.0.14.tar.gz | tar -xz \
docker/suse/Dockerfile:    && curl -fsSL https://download.libsodium.org/libsodium/releases/old/libsodium-1.0.14.tar.gz | tar -xz \
docker/ubuntu/Dockerfile:    && curl -fsSL https://download.libsodium.org/libsodium/releases/old/libsodium-1.0.14.tar.gz | tar -xz \
docs/build-environment.md:curl -fsSL https://github.com/jedisct1/libsodium/releases/download/1.0.14/libsodium-1.0.14.tar.gz | tar -xz
docs/build-environment.md:curl -fsSL https://github.com/jedisct1/libsodium/releases/download/1.0.14/libsodium-1.0.14.tar.gz | tar -xz
docs/build-environment.md:curl -fsSL https://github.com/jedisct1/libsodium/releases/download/1.0.14/libsodium-1.0.14.tar.gz | tar -xz
docs/build-environment.md:curl -fsSL https://github.com/jedisct1/libsodium/releases/download/1.0.14/libsodium-1.0.14.tar.gz | tar -xz

Reply:
Yes I can do that.